redbot/cogs/mod/settings.py

```diff
@@ -106,6 +112,24 @@ async def mentionspam(self, ctx: commands.Context):
         Manage the automoderation settings for mentionspam.
         """
 
+    @mentionspam.command(name="strict")
+    @commands.guild_only()
+    async def mentionspam_strict(self, ctx: commands.Context, toggle: bool):
+        """
+        Sets settings to allow for duplicate or only different mentions.
```

Comment:
Please reword this short-doc to be clearer.

Reply:
Reworded, not sure if the new one is better. Will wait for you to check.